projects/graphql-client/src/graphql-apollo-integration.test.ts

```diff
@@ -24,28 +24,40 @@ describe('GraphQl Apollo Integration Service', () => {
     ]
   });
 
-  const buildRequestString = (rootPath: string = 'test', params: string[] = []): string =>
+  const buildRequestString = (rootPath: string = 'test', params: string[] = [], fields: string[] = []): string =>
     new GraphQlRequestBuilder()
       .withSelects({
-        path: rootPath,
-        children: [{ path: 'id' }, { path: 'value' }],
-        ...params.map(param => ({ path: param }))
+        path: `${rootPath}${params.length > 0 ? `(${params.map(param => `${param}: ${param}`).join(',')})` : ''}`,
```

Comment:
By params I assume we mean arguments? Shouldn't they use the arguments portion of the selection?